src/main/resources/application.yaml

```diff
@@ -0,0 +1,87 @@
+#
+# JBoss, Home of Professional Open Source.
+# Copyright 2020-2022 Red Hat, Inc., and individual contributors
+# as indicated by the @author tags.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+quarkus:
+  log:
+    category:
+      org.jboss.pnc:
+        level: TRACE
+      org.elasticsearch:
+        level: INFO
+      org.apache.http:
+        level: INFO
+    console:
+      level: TRACE
+      format: '%d{HH:mm:ss,SSS} %-5p [%c] (%t) %s%e%n'
+      enable: 'true'
+    level: INFO
+  hibernate-orm:
+    database:
+      generation: validate
+  index-dependency:
+    elasticsearch:
+      artifact-id: elasticsearch
+      group-id: org.elasticsearch
+  http:
+    host: localhost
+    port: 8081
+  micrometer:
+    export:
+      json:
+        enabled: true
+  datasource:
+    db-kind: ${QUARKUS_DATASOURCE:postgresql}
+    jdbc.url: ${QUARKUS_DATASOURCE_URL:}
+    username: ${QUARKUS_DATASOURCE_USERNAME:}
+    password: ${QUARKUS_DATASOURCE_PASSWORD:}
+elasticsearch:
+  hosts: https://host:443
+  indexes: test-*
+  #keyStorePath =
+  #keyStorePass =
+  #keyPass =
+bifrost:
+  backoffrunnable:
+    delayMillis: 1000
+    maxBackOffCycles: 10
+    pollIntervalMillis: 250
```

Comment:
Does this mean that queries will be done every 250ms? In prod, this needs to be increased, as the query might take up to 1 second to execute

Reply:
ah ok, sounds good then. I would anyway go for 500 / 750, but if there are no concurrent executions, it's fine. thanks!